src/main/java/com/google/firebase/auth/internal/ListTenantsResponse.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.firebase.auth.internal;
+
+import com.google.api.client.util.Key;
+import com.google.firebase.auth.Tenant;
+import java.util.List;
+
+/**
+ * JSON data binding for ListTenantsResponse messages sent by Google identity toolkit service.
+ */
+public class ListTenantsResponse {
+
+  @Key("tenants")
+  private List<Tenant> tenants;
+
+  @Key("pageToken")
+  private String pageToken;
+
+  public ListTenantsResponse(List<Tenant> tenants, String pageToken) {
```

Comment:
You probably won't be able to parse/deserialize into this unless you also define a no-args constructor. Chances are the existing unit tests don't hit that case as of yet. But it's fine to defer adding it until we hit that case.

Reply:
Oh, you're right, I think we do need that (I slipped it into this PR).

No, there's not any unit tests which hit this yet. Those will come when I flesh out the other tenant methods in FirebaseUserManager.